### PR TITLE
[s3] revert skip deletion error, since the error file was not found is already skipped on the side of the grpc function

### PR DESCRIPTION
**Pull request by kmlebedev**
# What problem are we solving?

return an error when deleting a file

# How are we solving the problem?

There should not be a `filer_pb.ErrNotFound` error at this location
since it is already passed on the server side

https://github.com/seaweedfs/seaweedfs/blob/master/weed/server/filer_grpc_server.go#L296

# How is the PR tested?

# Checks
- [ ] I have added unit tests if possible.
- [ ] I will add related wiki document changes and link to this PR after merging.

### CODE REVIEW

File: weed/s3api/s3api_bucket_handlers.go [hunk 1]
```diff
@@ -346,7 +347,9 @@ func (s3a *S3ApiServer) GetBucketLifecycleConfigurationHandler(w http.ResponseWr
 			Expiration: Expiration{Days: days, set: true},
 		})
 	}
-
+	sort.Slice(response.Rules, func(i, j int) bool {
```
Comment (chrislusf) [inline weed/s3api/s3api_bucket_handlers.go]: this change seems not related.

Comment (kmlebedev) [reply weed/s3api/s3api_bucket_handlers.go]: > this change seems not related.

@chrislusf  Yes. I checked. Recently test_lifecycle_get started fluttering due to sorting.
reverted
